config/prometheus/alerts.yml

```
@@ -0,0 +1,553 @@
+
+      - alert: AlertmanagerConfigError
+        expr: alertmanager_config_last_reload_successful == 0
+        for: 1m
+        labels:
+          severity: high
+          service: alertmanager
+          team: ops
+        annotations:
+          summary: "Alertmanager configuration error"
+          description: "Alertmanager configuration reload failed"
+          runbook_url: "https://docs.wazuh-mcp.local/runbooks/alertmanager-config-error"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)